flexget/plugins/clients/deluge.py

```diff
@@ -317,8 +367,12 @@ def on_get_torrents_status(torrents):
             client.disconnect()
 
         filter = config.get('filter', {})
+
+        extra_keys = config.get('keys', [])
+        combined_list = self.settings_map.keys() + extra_keys
```

Comment:
hmm, guess it's not really needed, not sure why I did it like that in the first place.